app/gateway/changelog.md

```diff
@@ -753,6 +753,50 @@ This should be a 400 because the configuration is invalid.
 * When the OpenID Connect (OIDC) plugin is configured to reference HashiCorp Vault in the `config.client_secret` field (for example, `{vault://hcv/clientSecret}`),
 it does not look up the secret correctly.
 
+## 3.2.2.4
+**Release Date** 2023/08/21
+
+### Enterprise
+
+#### Breaking Changes
```

Comment:
❗ For reviewers: These were also added in the 3.4 changelog. Should I still leave them here because they were backported? If so, I think I'd also need to modify our breaking changes files.

Reply:
Good call. I would add a patches section in the breaking changes.